tests/aws/services/events/test_events.py

```diff
@@ -152,6 +152,27 @@ def test_put_event_without_detail_type(self, snapshot, aws_client):
         response = aws_client.events.put_events(Entries=entries)
         snapshot.match("put-events", response)
 
+    @markers.aws.validated
+    @pytest.mark.skipif(
+        is_old_provider(),
+        reason="V1 provider does not support this feature",
+    )
+    @pytest.mark.parametrize(
+        "detail",
+        ["NotJSON", "[]", "{{}", json.dumps("NotJSON")],
```

Comment:
nice also testing for plain string - was a bit surprised that this also results in malformed error but it is correct :)